app/services/proofing/aamva/response/authentication_token_response.rb

```diff
@@ -6,8 +6,8 @@ class AuthenticationTokenResponse
 
         def initialize(http_response)
           @http_response = http_response
-          handle_soap_error
           handle_http_error
+          handle_soap_error
           parse_response
```

Comment:
can we add a regression spec with an example payload showing why this is useful? ex `expect { }.to raise(VerificationError)`

Reply:
Added in [88f6d20](https://github.com/18F/identity-idp/pull/7735/commits/88f6d206d47039cc662b0bc13824b9ba401f8c27)